clang/lib/Parse/ParseDecl.cpp

```diff
@@ -3518,8 +3518,24 @@ void Parser::ParseDeclarationSpecifiers(
       DS.Finish(Actions, Policy);
       return;
 
-    case tok::l_square:
+    // alignment-specifier
+    case tok::kw__Alignas:
+      if (!getLangOpts().C11)
+        Diag(Tok, diag::ext_c11_feature) << Tok.getName();
```

Comment:
Preexisting but no compat warning in C11 mode?

Reply:
Yeah, we should probably add a compat warning here (and in a bunch of other places) -- perhaps a `good first issue`?